### DISCUSSION TITLE
How to make the background always #000000 and not the light black, for AMOLED?

### DISCUSSION THREAD

**Discussion by adamlahbib** [darkreader/darkreader#8573] [Q&A] [answered]
As the title indicates, websites like Google, Facebook, Twitter, YouTube, Reddit don't use the #000000 black background, how can I get the extension thru built in dev tools or something to switch background color as described for AMOLED screens.

**Comment by Gusted** [comment, marked as answer]
Click Dark Reader icon.
    Click Dev tools (in the bottom-right corner).
    Click Preview new design.
    Custom background, text and selection colors (under Theme click See all options, Colors).

**Reply by SpenHub** [reply]
~Clicking "preview new design" doesn't do anything on Firefox 116.0.3, Windows 11 22H2~
I'm an idiot, thought something was supposed to happen on devtools page. 

It changes the entire extension's UI. 
Full instructions: 
1. Open Extension
2. Click "Dev Tools" (Bottom right)
3. Click "Preview New Design"
4. Close out of menu, and re-open extension
5. Click: "See all options"
6. Click "Colors"
7. Modify `Custom Background`, `Text`, and `Selection` to your heart's content.

**Reply by MrPeteH** [reply]
Even then, it's not changing for me. And for some reason, everything has sepia even with sepia turned off. Frustrating.

**Reply by Lehmann11** [reply]
not a real workaround, but you can change websites like youtube and github to 150% contrast. it will make those pages backgrounds pitch black. had same problem when i switched to oled.
